webv2/helpers/helpers.go

```diff
@@ -37,12 +37,54 @@ const (
 	GOOGLE_SQL_DIALECT     = "Google Standard SQL"
 )
 
+var TABLE_STATEMENTS = []string{
+	`CREATE TABLE IF NOT EXISTS SchemaConversionSession (
+		VersionId STRING(36) NOT NULL,
+		PreviousVersionId ARRAY<STRING(36)>,
+		SessionName STRING(50) NOT NULL,
+		EditorName STRING(100) NOT NULL,
+		DatabaseType STRING(50) NOT NULL,
+		DatabaseName STRING(50) NOT NULL,
+		Dialect STRING(50) NOT NULL,
+		Notes ARRAY<STRING(MAX)> NOT NULL,
+		Tags ARRAY<STRING(20)>,
+		SchemaChanges STRING(MAX),
+		SchemaConversionObject JSON NOT NULL,
+		CreateTimestamp TIMESTAMP NOT NULL,
+	) PRIMARY KEY(VersionId)`,
+	`CREATE TABLE IF NOT EXISTS SMT_JOBS (
+		JobId STRING(100) NOT NULL,
+		JobName STRING(100) NOT NULL,
+		JobType STRING(100) NOT NULL,
+		JobData JSON,
+		Dialect STRING(50) NOT NULL,
+		SpannerDatabaseName STRING(100) NOT NULL,
+		CreatedAt TIMESTAMP NOT NULL,
+	) PRIMARY KEY(JobId)`,
+	`CREATE TABLE IF NOT EXISTS SMT_RESOURCES (
+		ResourceId STRING(100) NOT NULL,
+		JobId STRING(100) NOT NULL,
+		ExternalId STRING(100) NOT NULL,
+		ResourceName STRING(100) NOT NULL,
+		ResourceType STRING(100) NOT NULL,
+		ResourceData JSON,
+		CreatedAt TIMESTAMP NOT NULL,
+	) PRIMARY KEY(ResourceId)`,
+	`CREATE TABLE IF NOT EXISTS SMT_STATES (
+		StateId STRING(100) NOT NULL,
+		StateVersion INT64 NOT NULL,
+		ResourceId STRING(100) NOT NULL,
+		StateData JSON,
+		CreatedAt TIMESTAMP NOT NULL,
+	) PRIMARY KEY(StateId, StateVersion)`,
+}
+
 func GetSpannerUri(projectId string, instanceId string) string {
 	return fmt.Sprintf("projects/%s/instances/%s/databases/%s", projectId, instanceId, constants.METADATA_DB)
 }
 
 // Creates the schema for the internal metadata database
-func createDatabase(ctx context.Context, uri string) error {
+func createDatabase(ctx context.Context, uri string, isExist bool) error {
```

Comment:
nit: better to rename isExist to dbExists

Reply:
Done